examples/vertx-opentracing/src/main/java/org/hawkular/apm/examples/vertx/opentracing/InventoryManager.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.apm.examples.vertx.opentracing;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
+import io.opentracing.Tracer;
+import io.opentracing.propagation.Format;
+import io.vertx.core.eventbus.EventBus;
+import io.vertx.core.eventbus.Message;
+import io.vertx.core.eventbus.MessageConsumer;
+import io.vertx.core.json.JsonObject;
+
+/**
+ * @author gbrown
+ */
+public class InventoryManager {
+
+    private Map<String, JsonObject> items = new HashMap<>();
+    private MessageConsumer<JsonObject> getItemConsumer;
+    private MessageConsumer<JsonObject> ordersConfirmedConsumer;
+
+    private InventoryManager(EventBus eb, Tracer tracer) {
+        initItems();
+        init(eb, tracer);
+    }
+
+    public static InventoryManager create(EventBus eb, Tracer tracer) {
+        return new InventoryManager(eb, tracer);
+    }
+
+    protected void initItems() {
+        items.put("laptop", new JsonObject().put("itemId", "laptop").put("quantity", 5));
+        items.put("car", new JsonObject().put("itemId", "car").put("quantity", 10));
+    }
+
+    protected void init(EventBus eb, Tracer tracer) {
```

Comment:
I forgot this one, can be this also broken into two methos (`getItemConsumer`, ` ordersConfirmedConsumer`)